virttest/qemu_monitor.py

```diff
@@ -150,13 +150,14 @@ def create_monitor(vm, monitor_name, monitor_params):
             MonitorClass = QMPMonitor
 
     logging.info("Connecting to monitor '<%s> %s'", MonitorClass, monitor_name)
-    monitor = MonitorClass(vm, monitor_name, monitor_params)
+    monitor = MonitorClass(vm, monitor_name, monitor_params, suppress_exceptions)
     monitor.verify_responsive()
 
     return monitor
 
 
-def wait_for_create_monitor(vm, monitor_name, monitor_params, timeout):
+def wait_for_create_monitor(vm, monitor_name, monitor_params,
+                            suppress_exceptions, timeout):
```

Comment:
Please append the new parameter to the end.

Reply:
@vivianQizhu  You are right, i understand your point, to avoid this change effect other calling.
I checked tp-libvirt, tp-spice, and tp-qemu, no call for this function. The only one call is in `qemu_vm.py` which i have update it.  Prefer to put timeout at the end of the parameter a bit more, Is it acceptable for you ?

Reply:
@yanan-fu Yes it's also okay for me.

Reply:
Thanks~